src/ledger/transaction.ts

```diff
@@ -325,6 +325,14 @@ export const settlePayment = async (hash) => {
   return result.nModified > 0
 }
 
+export const settleLndPayment = (hash) => {
+  return settlePayment(hash)
+}
+
+export const settleOnchainPayment = (hash) => {
+  return settlePayment(hash)
```

Comment:
I guess both lightning and onchain tx have a hash? let me dive back into it.

Reply:
I think its ok, if you see we have differnet methods for onchain and lightning... but let me know how is better for you

Reply:
I didn't had in mind both lightning and onchain are both using a hash and so we could have a unique settlePayment method here. we can leave it as is.
I'm not sure this is a best practice overall, because those hash represent different things. one is hash of a pre-image, the other is hash of a onchain transaction

Reply:
Yes, having consolidated methods in the ledger module helps to see things like this. I think we must keep this in mind if we move from db/medici